src/sgame/sg_bot_skilltree.cpp

```diff
@@ -24,6 +24,44 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 */
 
 #include "sg_bot_util.h"
+#include "../shared/parse.h"
+
+// aliens have 61 points to spend max
+// humans have 40 points to spend max
+Cvar::Cvar<int> g_skillsetBudgetAliens( "g_skillsetBudgetAliens", "the skillset budget for aliens.", Cvar::NONE, 61 );
+Cvar::Cvar<int> g_skillsetBudgetHumans( "g_skillsetBudgetHumans", "the skillset budget for humans.", Cvar::NONE, 40 );
```

Comment:
I would make them callback cvars too which would call `G_UpdateSkillsets` (see the other comment) on update. This would make the changes live.